experimental/add_to_app/README.md

```diff
@@ -16,7 +16,22 @@ a standalone Flutter module called `example_module`.
 ### `example_module`
 
 This is the Flutter module that'll be added to all the individual projects
-in the `add2app` folder.
+in the `add_to_app` folder.
+
+### `SimpleAndroidApp`
+
+An Android application that imports `example_module` and uses it to display
+a `FlutterActivity` containing widgets from the module. This project brings
+in the Flutter module's Android host wrapper as a gradle sub-module. To set
+it up prior to running it for the first time:
+
+```bash
+cd example_module
+flutter packages get
+```
+
+Then open the SimpleAndroidApp folder in Android Studio. Gradle with sync the
```

Comment:
I'm not able to parse "Gradle with sync the Flutter module in as a sub-module." Is there a transposition in there, or am I just not reading it right?

Reply:
ah brain melt. Revised